scala3doc/src/dotty/dokka/model/api/api.scala

```diff
@@ -124,6 +126,7 @@ extension[T] (member: Member):
 
   def signature: Signature = memberExt.fold(Signature(name))(_.signature)
   def asLink: LinkToType = LinkToType(signature, dri, kind)
+  def deprecated: Option[Annotation] = memberExt.flatMap(_.annotations.find(a => a.dri.getPackageName == "scala" && a.dri.getClassNames == "deprecated"))
```

Comment:
What if we are have a deprecated member from Java?

Reply:
It depends. It still will be recognized as an annotation, though it won't have added strikethrough style. The question is: if we really find that using `java.lang.Deprecated` in Scala code should cause any special treatment. In my opinion it's user fault

Reply:
What if one want to use a Java class with deprecated members (e.g. [this one](https://docs.oracle.com/en/java/javase/13/docs/api/java.desktop/javax/swing/JComponent.AccessibleJComponent.html) )

Reply:
You mean as a base class for Scala class... Yeah, we would like to make the Inherited methods deprecated. Will add support then

Reply:
I think this can be done in follow up PR